src/config.js

```diff
@@ -0,0 +1,156 @@
+/*
+ * Module for getting and setting Prebid configuration.
+ *
+ * Prebid previously defined these properties directly on the global object:
+ * pbjs.logging = true;
+ *
+ * Defining and access properties in this way is now deprecated, but these will
+ * continue to work during a deprecation window.
+ */
+const utils = require('./utils');
+
+const DEFAULT_DEBUG = false;
+const DEFAULT_BIDDER_TIMEOUT = 3000;
+const DEFAULT_PUBLISHER_DOMAIN = window.location.origin;
+const DEFAULT_COOKIESYNC_DELAY = 100;
+const DEFAULT_ENABLE_SEND_ALL_BIDS = false;
+
+const ALL_TOPICS = '*';
+
+let listeners = [];
```

Comment:
I don't think this would really gain us anything.  You would have to move the `let config` into the `newConfig` closure as well, right?  So then we still have a "global" state for Prebid.js, it's just inside of a closure now making it harder to get at.  Also, this means `getConfig`, `setConfig`, etc would need to be updated to pass in the closure reference to that state so they could interact with it.  

```
import { newConfig } from './config';

const config = newConfig();
$$PREBID_GLOBAL$$.getConfig = config.getConfig
$$PREBID_GLOBAL$$.setConfig = config.setConfig
```
Wouldn't each of the tests have to do this as well in order to reset global Prebid.js state?  I think i'd be a little more straight forward to have a `resetConfig()` that clears any config state, in this case `config` and `listeners`.